build_openroad.sh

```diff
@@ -251,8 +251,8 @@ __docker_build()
                 cp .dockerignore{,.bak}
                 sed -i '/flow\/platforms/d' .dockerignore
         fi
-        ./etc/DockerHelper.sh create -target=dev -os="${DOCKER_OS_NAME}"
-        ./etc/DockerHelper.sh create -target=builder -os="${DOCKER_OS_NAME}"
+        ./etc/DockerHelper.sh create -target=dev -os="${DOCKER_OS_NAME}" -threads="${PROC}"
+        ./etc/DockerHelper.sh create -target=builder -os="${DOCKER_OS_NAME}" -threads="${PROC}"
```

Comment:
`-threads` automatically taken by script. Only if user needs to restrict the number of threads need to use the arguments

Reply:
I understand. The reason I added it in is people might fail at the openroad stage because of insufficient threads [1]. etc/DockerHelper.sh defaults to max threads, which is whats causing the failure. 

Also I don't think there are any side effects for adding this line, since PROC will definitely have a value inside ./build_openroad.sh script

[1]: https://github.com/The-OpenROAD-Project/OpenROAD/issues/3366

Reply:
I agree with @luarss, it is correct to add it here.